### PR TITLE
Update README to include 2.0.0 links

### PR DESCRIPTION
**Pull request by karajas**
skip ci please

@Petermarcu @gkhanna79 @eerhardt @chcosta @leecow @weshaggard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +67,19 @@ Daily Builds
 [win-x64-badge-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/sharedfx_win-x64_Release_version_badge.svg
 [win-x64-version-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/latest.sharedfx.win.x64.version
 [win-x64-installer-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.exe
+[win-x64-installer-checksum-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/dotnet-win-x64.latest.exe
 [win-x64-zip-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.zip
+[win-x64-zip-checksum-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.zip.sha512
 [win-x64-symbols-zip-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-sharedframework-symbols-win-x64.latest.zip
 
+[win-x64-badge-2.X.X]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/sharedfx_win-x64_Release_version_badge.svg
+[win-x64-version-2.X.X]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/latest.sharedfx.win.x64.version
```
Comment (eerhardt) [inline README.md]: This link doesn't exist.

Can you manually ensure each link you are adding works?

File: README.md [hunk 2]
```diff
@@ -30,30 +30,33 @@ They can be downloaded from [here](https://www.microsoft.com/net/download#core).
...
+|   **Fedora 24 (x64)**      |   N/A   |   N/A   |   [![][fedora-24-badge-1.1.X]][fedora-24-version-1.1.X]<br>[tar.gz][fedora-24-targz-1.1.X]   |   N/A   |
+|   **OpenSUSE 42.1 (x64)**  |   N/A   |   N/A   |   [![][opensuse-42.1-badge-1.1.X]][opensuse-42.1-version-1.1.X]<br>[tar.gz][opensuse-42.1-targz-1.1.X]   |   N/A   |
 
 *Note: Our .deb packages are put together slightly differently than the other OS specific installers. Instead of combining everything, we have separate component packages that depend on each other. If you're installing these directly from the .deb files (via dpkg or similar), then you'll need to install them in the order presented above.*
 
 [build-badge-master]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/3160/badge
 [build-master]: https://devdiv.visualstudio.com/DevDiv/_build/index?definitionId=3160&_a=completed
 
+[build-badge-2.X.X]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/6161/badge
```
Comment (eerhardt) [inline README.md]: should be `build-badge-2.0.X`.  When we have a 2.1 branch, they need to be separate.

File: README.md (hunk 1 above)
Comment (karajas) [inline README.md]: Sure can I merge the checksum PR and kick off another build?

File: README.md (hunk 2 above)
Comment (karajas) [inline README.md]: Cool will update.

**Pull request merged**